source/manage-apps/configure/config.txt

```diff
@@ -38,6 +38,15 @@ using structured JSON configuration and JavaScript source code files.
 - ``.js`` files define serverless application logic used in functions, triggers,
   HTTPS endpoints, and custom resolvers.
 
+.. important:: Reserved File Names
+   
+   From a CLI perspective, there are a few special/reserved file names we advise
+   you to avoid using for your own purposes: ``stitch.json``, ``config.json``, 
```

Comment:
can you phrase this more directly? 

you want to convey the following ideas here i think:
* you can include arbitrarily named files in your realm app config
* with the exception of these specific file names, as they are reserved for config files
* including improperly formatted files with these names causes the Realm CLI to throw an error when you import them